libraries/passive_sensitive.rb

```diff
@@ -0,0 +1,13 @@
+# coding: UTF-8
+#
+# Cookbook Name:: cerner_splunk
+# File Name:: passive_sensitive.rb
+
+require 'chef/resource'
+
+class Chef
+  class Resource
+    def sensitive(arg)
```

Comment:
Updated per your recommendation. I kinda get why we want it to match the real definition. As I can come up with a few (some contrived) reasons for matching and can't come up with any reasons not to, I made the change :p